### ISSUE TITLE
Rec. Training: learning rate

### ISSUE THREAD

**Issue by jbarth-ubhd** [mittagessen/kraken#747]
https://kraken.re/main/training/rectrain.html#best-practices

> ... to scale the learning rate by the square root of the batch size (1e-3 * sqrt(batch_size)).

To my experience, a learning rate (`-r`) >> 0.001 results in massive drop of the accuracy (or massive increase of training_loss) sooner or later.

Tried e. g. with `-B 36 -r 0.006` and with `-B 36 -r 0.003 # more conservative`.

**Comment by jbarth-ubhd** [comment]
PS: catmus-medieval has lrate: 0.**000**1 and batch_size: 64
